src/setup.rs

```diff
@@ -18,19 +18,31 @@ const VERSION: &str = env!("CARGO_PKG_VERSION");
 /// Set by our `build.rs`, reflects the Rust target triple we're building for
 const TARGET: &str = env!("TARGET");
 
-/// Where Kani has been installed. Typically `~/.kani/kani-1.x/`
-pub fn kani_dir() -> PathBuf {
-    home::home_dir().expect("Couldn't find home dir?").join(".kani").join(format!("kani-{VERSION}"))
+/// The directory where Kani is installed, either:
+///  * `${KANI_HOME}/.kani/kani-<VERSION>` if the environment variable
```

Comment:
As a user, I would expect `${KANI_HOME}/kani-<version>`. I.e., I wouldn't expect `.kani` to be inside the set `$KANI_HOME`. That is how `CARGO_HOME`, `RUSTUP_HOME`, and other tools usually work.

Reply:
Makes a lot of sense, thanks! Changed to add "/.kani" on the `HOME` arm only.